lib/site_setting_extension.rb

```diff
@@ -523,6 +523,10 @@ def info(name)
     end
   end
 
+  def valid_areas
+    SiteSetting::VALID_AREAS.concat(DiscoursePluginRegistry.site_setting_areas.to_a).uniq
```

Comment:
I think since the `DiscoursePluginRegistry.site_setting_areas` is a `Set`, you can use a `Set` here too like I did for bookmarks:

https://github.com/discourse/discourse/blob/d2dce328b86130dbf1efa780a8fd3425664e63da/app/models/bookmark.rb#L9-L11